dist/common/scripts/scylla-housekeeping

```diff
@@ -124,6 +124,13 @@ def get_repo_file(dir):
                     return match.group(2), match.group(1)
     return None, None
 
+def get_env_type(name):
+    if os.path.exists(name):
+        with open(name, 'r') as myfile:
+            for line in myfile:
```

Comment:
why do you iterate over the lines? strip() method removes all the whitespace chars (including newlines)

The code can be written using Python3:

```python
get_env_type(platform_file_path=Path(args.platform_file))
...
def get_env_type(platform_file_path):
    if platform_file_path.exists():
        with platform_file_path.open() as file:
            if platform := file.read().strip():
                return platform
    return None

Reply:
I'll update the code I'm not sure we can use the walrus operator, it was only added in 3.8

Reply:
I've forced push an update (as I've mentioned without the walrus operator, it's cool but there are too many 3.6 and 3.7 out there)